presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -257,7 +255,7 @@ public ThriftHiveMetastoreStats getStats()
     @Override
     public Set<ColumnStatisticType> getSupportedColumnStatistics(MetastoreContext metastoreContext, Type type)
     {
-        return MetastoreUtil.getSupportedColumnStatistics(type);
+        return ThriftMetastoreUtil.getSupportedColumnStatistics(type);
```

Comment:
I would inline the function here instead of adding it to Util since it's not used anywhere else.